velocidi-sdk/src/main/kotlin/com/velocidi/HttpClient.kt

```diff
@@ -0,0 +1,63 @@
+package com.velocidi
+
+import android.util.Log
+import com.android.volley.RequestQueue
+import com.android.volley.Response
+import com.android.volley.toolbox.*
+import java.io.File
+import com.android.volley.toolbox.StringRequest
+import org.json.JSONObject
+import java.nio.charset.Charset
+
+
+class HttpClient {
```

Comment:
We opted to create our own http client utility to minimize the lib and because this was simple?

Can we consider making this a bit less mutable, maybe passing on the headers and default params as arguments?

Reply:
Done in 2a872bc.

On a side note, we need to handle HTTP redirect requests ourselves. `Volley` doesn't support it.